src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -426,6 +510,26 @@ $DOCKERFILE_SIGIL
     imageDirectory
   }
 
+  /** Task to build the main docker image for the project. This returns the image ID. */
+  lazy val mainImageBuildDef: Def.Initialize[Task[String]] = Def.task {
+    // This task requires docker to be installed.
+    requireDocker.value
+
+    // This task requires that the dependency image be created, and that the main image be staged.
+    mainImageStageDef.value
+    dependencyBuildDef.value
+
+    val logger = Keys.streams.value.log
+    logger.info("Building main image...")
```

Comment:
There might be a way to make this work simply using sbt's builtins. I'll play around a little tomorrow & send you another PR once I get it working.